tests/subsys/portability/cmsis_rtos_v2/src/event_flags.c

```diff
@@ -217,15 +216,9 @@ void test_event_flags_isr(void)
 	zassert_true(osEventFlagsDelete(evt_id) == osOK,
 		     "EventFlagsDelete failed");
 }
-ZTEST(cmsis_event_flags, test_event_flags)
+void cmsis_event_flags_before(void *data)
 {
-	/*
-	 *These tests are order-dependent.
-	 *They have to be executed in order.
-	 *So put these tests in one ZTEST.
-	 */
-	test_event_flags_no_wait_timeout();
-	test_event_flags_signalled();
-	test_event_flags_isr();
+	ARG_UNUSED(data);
```

Comment:
There test are removed? Looks incorrect for me.

Reply:
Before I made these tests into a ZTEST, because they were dependent. But now I get rid of dependencies of tests, so I removed these from a ZTEST.

Reply:
Ok, I saw you fixed it. thanks.